locales/ach/server.properties

```diff
@@ -24,6 +24,7 @@ pageTitleFeatures=Thimble by Mozilla - Jami
 
 # Userbar
 signInPromptHomepage=<a href="#" title="Sign in to Thimble" id="login-link" data-loginUrl="{{ loginURL }}" class="navbar-login">Dony iyie</a> onyo <a href="#" title="Create an account" id="signup-link" class="navbar-login">Cwe akaunt</a>
+signInPromptHomepage2=<a href="#" title="Sign in to Thimble" id="login-link" data-loginUrl="{{ loginURL }}" class="navbar-login">Dony iyie</a>
```

Comment:
Do these still get updated automatically? (with the repo archived, I assume "no"). If not, do we need to get all of these locale strings confirmed before we land this? (not every language has grammar in which a subclause and that clause as full sentence use the same words, conjugations, and inflections...)